app/views/sessions/new.html.erb

```diff
@@ -1,12 +1,22 @@
-<h1>Login</h1>
-<%= form_tag sessions_path do %>
-  <div class="field">
-    <%= label_tag :email %>
-    <%= text_field_tag :email %>
+<div id="login">
+  <div class="contiainer">
+    <div class ='col-xs-12 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 well'>
```

Comment:
from fer

>fix typo, also container needs inside a row, and a row can have many col-size-*, then you insert the content. That way it will stay responsive across sizes, and you can always create another container inside the view, it will just add a bit more padding